include/wx/msw/listctrl.h

```diff
@@ -383,6 +383,9 @@ class WXDLLIMPEXP_CORE wxListCtrl: public wxListCtrlBase
                                     WXWPARAM wParam,
                                     WXLPARAM lParam) wxOVERRIDE;
 
+    // check if a native double-buffering applies for this window
```

Comment:
I don't think it's really worth copying the comment from the base class virtual method in all classes where it's overridden.